src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/AutoFixture/AutoDataWithoutChildsAttribute.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Linq;
+using AutoFixture;
+using AutoFixture.Xunit2;
+
+namespace DevAdventCalendarCompetition.Tests.AutoFixture
+{
+    [AttributeUsage(AttributeTargets.Method)]
+    public sealed class AutoDataWithoutChildsAttribute : AutoDataAttribute
```

Comment:
I've had a problem with correct naming, what do you think?

Reply:
I suggest `TestWithNumberWithoutAnswersData`